crates/next-custom-transforms/tests/fixture/server-actions/server/8/output.js

```diff
@@ -1,4 +1,4 @@
-/* __next_internal_action_entry_do_not_use__ {"706a88810ecce4a4e8b59d53b8327d7e98bbf251d7":"$$RSC_SERVER_ACTION_0"} */ import { registerServerReference } from "private-next-rsc-server-reference";
+/*#__TURBOPACK_DISABLE_EXPORT_MERGING__*/ /* __next_internal_action_entry_do_not_use__ {"706a88810ecce4a4e8b59d53b8327d7e98bbf251d7":"$$RSC_SERVER_ACTION_0"} */ import { registerServerReference } from "private-next-rsc-server-reference";
```

Comment:
Why is this comment on top level and not on the export?

Reply:
I think this is a bug of `swc_ecma_codegen`.